docs/reference/authenticated_authors.rst

```diff
@@ -12,19 +12,14 @@ This is a ``FOSCommentBundle`` feature. We will explain to you here how to enabl
 Implement SignedCommentInterface
 --------------------------------
 
-Open entity ``Application\Sonata\CommentBundle\Entity\Comment`` and implement the following methods and interface:
+Open ``App\\Entity\Comment`` and implement the following methods and interface::
 
-.. code-block:: php
-
-    <?php
-
-    namespace Application\Sonata\CommentBundle\Entity;
+    namespace App\Entity;
 
     use FOS\CommentBundle\Model\SignedCommentInterface;
+    use Sonata\CommentBundle\Entity\BaseComment;
     use Symfony\Component\Security\Core\User\UserInterface;
 
-    use Sonata\CommentBundle\Entity\BaseComment as BaseComment;
-
     class Comment extends BaseComment implements SignedCommentInterface
```

Comment:
Should we use annotations for the entity classes?

Reply:
Not yet